.github/dependabot.yml

```diff
@@ -1,25 +1,29 @@
 version: 2
 updates:
-  - package-ecosystem: "gomod"
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 5
-    ignore:
-    - dependency-name: "k8s.io/*"
-    - dependency-name: "sigs.k8s.io/*"
-    - dependency-name: "github.com/containernetworking/*"
-    - dependency-name: "github.com/k8snetworkplumbingwg/*"
-      update-types: ["version-update:semver-major", "version-update:semver-minor"] # ignore all except for patch updates
-    - dependency-name: "github.com/vmware/go-ipfix"
-    - dependency-name: "github.com/TomCodeLV/OVSDB-golang-lib"
-    - dependency-name: "github.com/aws/*" # updates are too frequent
-    - dependency-name: "antrea.io/ofnet"
-    - dependency-name: "antrea.io/libOpenflow"
-    - dependency-name: "github.com/ClickHouse/clickhouse-go/v2" # auto-upgrade involves dependency conflicts
-  - package-ecosystem: "github-actions"
+- package-ecosystem: "gomod"
+  directory: "/"   # Location of package manifests
+  schedule:
+    interval: "daily"
+  open-pull-requests-limit: 5
+  ignore:
+  - dependency-name: "k8s.io/*"
+  - dependency-name: "sigs.k8s.io/*"
+  - dependency-name: "github.com/containernetworking/*"
+  - dependency-name: "github.com/k8snetworkplumbingwg/*"
+    update-types: ["version-update:semver-major", "version-update:semver-minor"]   # ignore all except for patch updates
+  - dependency-name: "github.com/vmware/go-ipfix"
+  - dependency-name: "github.com/TomCodeLV/OVSDB-golang-lib"
+  - dependency-name: "github.com/aws/*"   # updates are too frequent
+  - dependency-name: "antrea.io/ofnet"
+  - dependency-name: "antrea.io/libOpenflow"
+  - dependency-name: "github.com/ClickHouse/clickhouse-go/v2"   # auto-upgrade involves dependency conflicts
+- package-ecosystem: "github-actions"
     # Workflow files stored in the default location of `.github/workflows`
-    directory: "/"
-    schedule:
-      interval: "daily"
-    open-pull-requests-limit: 5
+  directory: "/"
+  schedule:
+    interval: "daily"
+  open-pull-requests-limit: 5
+- package-ecosystem: docker
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
Thanks @zhuxiaow0, but I'm not sure if this is wanted in antrea as we don't have many base images to manage and versions of most base images are even parameterized so I guess the tool can't even know which versions are used exactly. The only one that is not parameterized is ubuntu image, which will likely bump up every two years and each bumpping up may need human intervention according to previous experience: https://github.com/antrea-io/antrea/pull/4459.
but I would defer to @antoninbas who is more familiar with this.